rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,31 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize(serializer: nil)
+        @serializer = serializer || ::YAML
+      end
+
+      def build_event_payload(event)
```

Comment:
Things I would consider changing:

* method name
* Using an object (instead of open Hash) or ClosedStruct
  * or using 4 methods such as `event_id(event)`, `binary_data(event)`, etc

Reply:
any suggestion about method names are welcome,

about object creation, which way you prefer? I'm open for suggestion